src/NServiceBus.Core/Recoverability/FirstLevelRetries/ProcessingFailureInfo.cs

```diff
@@ -0,0 +1,16 @@
+﻿namespace NServiceBus
+{
+    using System;
+
+    class ProcessingFailureInfo
```

Comment:
Not convinced that we should reuse this one for the TM, I'd say we try to make the TM recoverability completely autonomous?